order-service/src/test/java/com/learningmicroservices/orderservice/OrderServiceIntegrationTest.java

```diff
@@ -0,0 +1,125 @@
+package com.learningmicroservices.orderservice;
+import com.learningmicroservices.orderservice.Enum.PaymentStatus;
+import com.learningmicroservices.orderservice.dto.InventoryResponse;
+import com.learningmicroservices.orderservice.dto.OrderLineItemsDto;
+import com.learningmicroservices.orderservice.dto.OrderRequest;
+import com.learningmicroservices.orderservice.event.OrderPlacedEvent;
+import com.learningmicroservices.orderservice.model.Order;
+import com.learningmicroservices.orderservice.repository.OrderRepository;
+import com.learningmicroservices.orderservice.service.OrderService;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.kafka.core.KafkaTemplate;
+import org.springframework.web.reactive.function.client.WebClient;
+import reactor.core.publisher.Mono;
+
+import java.math.BigDecimal;
+import java.util.Collections;
+import java.util.UUID;
+import java.util.function.Function;
+
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.*;
+import static org.junit.jupiter.api.Assertions.*;
+
+@ExtendWith(MockitoExtension.class)
```

Comment:
integration test are @springbootest  .. .extendwith is a plain junit. integration test dont have mocks